core/src/limits.rs

```diff
@@ -67,25 +42,15 @@ impl ExecutionLimit {
     ///
     /// This is intended to be called after the given operations have been
     /// executed. The ops will be deducted from the operation limit and, if
-    /// that limit is zero, the time limit will be checked. If both limits have
-    /// been breached, this returns `true`. Otherwise, this returns `false`,
-    /// and if the operation limit was exhausted, it will be returned to the
-    /// starting maximum.
+    /// that limit has been breached, this returns 'true'
     pub fn did_ops_breach_limit(
         &mut self,
-        context: &mut UpdateContext<'_, '_>,
+        _context: &mut UpdateContext<'_, '_>,
```

Comment:
Can the `context` parameter can be removed from here now?

Reply:
Note that there's been quite some discussion about the point or validity of this PR on Discord.